ocaml/idl/datamodel_vtpm.ml

```diff
@@ -61,7 +64,10 @@ let t =
         (Published, rel_rio, "Added VTPM stub")
       ; (Extended, rel_next, "Added ability to manipulate contents")
       ; (Extended, rel_next, "Added VTPM profiles")
-      ; (Changed, rel_next, "Removed backend field")
```

Comment:
I think we should keep the 'backend' field removed (it used to mean "domain" and had a different type), might avoid issues with DB schema upgrade.

And you've already called the backend 'persistence_backend' above in the type, so we should make the field name match as well.